src/bcrypt/__init__.py

```diff
@@ -125,6 +126,14 @@ def kdf(password, salt, desired_key_bytes, rounds):
     if rounds < 1:
         raise ValueError("rounds must be 1 or more")
 
+    if rounds < 50 and not ignore_few_rounds:
+        # They probably think bcrypt.kdf()'s rounds parameter is logarithmic,
+        # expecting this value to be slow enough (it probably would be if this
+        # were bcrypt). Emit a warning.
+        sys.stderr.write(("Warning: bcrypt.kdf() called with only {} round(s). "
```

Comment:
You can do this with `warnings.warn` (https://docs.python.org/2/library/warnings.html). You can see an example here: https://github.com/pyca/cryptography/blob/a5865d9b630df7586785b683ed145b977ad7e0c4/src/cryptography/__init__.py#L22-L27

That example uses `DeprecationWarning`, but `UserWarning` is more appropriate here.